src/os/bluestore/BlueStore.cc

```diff
@@ -6138,8 +6134,6 @@ int BlueStore::_do_zero(TransContext *txc,
   // they may touch.
   o->flush();
 
-  o->bc.discard(offset, length);
```

Comment:
No replacement for that? Since now you are relying on the lextent map to bypass cache usage for holes, right? IMHO that might be dangerous, wouldn't that bring some hidden issue? As for me I'd prefer to have consistent cache content too.